schemas/stsci.edu/asdf/core/externalarray-1.0.0.yaml

```diff
@@ -20,7 +20,7 @@ examples:
 
 type: object
 properties:
-  source:
+  fileuri:
```

Comment:
Does this require a corresponding change in asdf?

Reply:
No, it was correct in ASDF but since required properties weren't being validated correctly, and since `additionalProperties` is set to `true`, the error in the schema wasn't picked up.